docs/public/static/diffs/expo-ios.diff

```diff
@@ -63,4 +67,171 @@ index 99b1b8d..71524e2 100644
 +      Pod::UI.warn e
 +    end
 +  end
-   config = use_native_modules!
\ No newline at end of file
+   config = use_native_modules!
+diff --git a/ios/MyApp.xcodeproj/project.pbxproj b/ios/MyApp.xcodeproj/project.pbxproj
+index df60006..e5ee03a 100644
```

Comment:
@Kudo, I wasn't sure if we need to include the diff from this file, so it's included for now. Let me know if this is the case or not and we should remove this file changes. In the previous change, this wasn't included.

Reply:
it's not necessary. we could remove the project.pbxproj changes. thanks @amandeepmittal!

Reply:
Thanks for your input! Fixed!